### DISCUSSION TITLE
Custom extension open always in single windows

### DISCUSSION THREAD

**Discussion by tb56zily** [microsoft/vscode-discussions#931] [Extension Development QnA]
Hello Team,

I have my custom vs extension that loads webview (blazor) for a set of specific xml files. When i open one of these xmls, my extension is loaded and works as expected. But when i open another xml, already opened webview gets replaced. The only problem I am stuck at is opening my extension in multiple tabs for multiple documents. 
I tried setting **supportsMultipleEditorsPerDocument = true** but no luck.

I need your support in this. Thanks in Advance!

**Comment by burkeholland** [comment]
Are  you creating a new WebViewPanel each time? https://code.visualstudio.com/api/references/vscode-api#window.createWebviewPanel

**Reply by olguzzar** [reply]
Hi @tb56zily, did the above documentation help with what you're trying to do? If not, can you please provide some more information? Thanks!

**Reply by tb56zily** [reply]
Hi Bruke, 
Thanks for the response. I am creating new WebViewPanel each time.

**Comment by abrilurena** [comment]
Hi, @tb56zily! It's been a while, I just wanted to check in and see if you still need any help with this?

**Comment by burkeholland** [comment]
I'm closing this item for now since there hasn't been an update since last year. Feel free to re-open if you feel this is an error.
